UnitsNet/BaseUnits.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Copyright (c) 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com).
+// https://github.com/angularsen/UnitsNet
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+using System;
+using System.Text;
+using UnitsNet.Units;
+
+namespace UnitsNet
+{
+#if !WINDOWS_UWP
+    public sealed partial class BaseUnits : IEquatable<BaseUnits> { }
+#endif
+
+    /// <summary>
+    ///     Represents the base units for a quantity. All quantities, both base and derived, can be
+    ///     represented by a combination of these seven base units.
+    /// </summary>
+    public sealed partial class BaseUnits
+    {
+        /// <summary>
+        /// Creates an instance of if the base units class that represents the base units for a quantity.
+        /// All quantities, both base and derived, can be represented by a combination of these seven base units.
+        /// </summary>
+        /// <param name="length">The length unit (L).</param>
+        /// <param name="mass">The mass unit (M).</param>
+        /// <param name="time">The time unit (T).</param>
+        /// <param name="current">The electric current unit (I).</param>
+        /// <param name="temperature">The temperature unit (Θ).</param>
```

Comment:
Just a heads up, I had trouble with this character `Θ` in powershell. Had to use `P` instead there, or at least I couldn't figure it out at the time.

Reply:
Where is it used in PowerShell?

Reply:
Parsing base dimensions from JSON, then passing on its value to generated c# code.